### PR TITLE
Created BDD test for list al items

### PR DESCRIPTION
**Pull request by higuchihisao**

### CODE REVIEW

File: features/shopcarts.feature [hunk 1]
```diff
@@ -75,4 +75,12 @@ Scenario: Add an item to a shopcart, and then delete the entire shopcart
     Then I should see the message "Success! Added 5 iPod MAX to this shopcart"
     When I paste the "Shopcart_ID_top" field
     And I press the "Delete" button
-    Then I should see the message "Shopcart deleted"
\ No newline at end of file
+    Then I should see the message "Shopcart deleted"
+
+Scenario: List all items in a shopcart
+    When I visit the "Home Page"
+    And I set the "Shopcart_ID_bottom" to "1"
```
Comment (rgordon126) [inline features/shopcarts.feature:82]: You need to actually add a shopcart and items here. Look to lines 60-74 to copy and paste and edit the field names as you see fit.

**Review by rgordon126 — CHANGES_REQUESTED** [review]
See edits. The BDD test fails.

File: features/shopcarts.feature (hunk 1 above)
Comment (higuchihisao) [reply features/shopcarts.feature:82]: I saw the issue, and it only mentions that we are testing showing the items in the shopcart. I think there is another story for adding an item to a shopcart.

Comment (rgordon126) [reply features/shopcarts.feature:82]: Yes I already did that story. You just need to copy the code so you can test your story. If you run behave on your story it fails. More details in the slack.

Comment (higuchihisao) [reply features/shopcarts.feature]: Now is ready to merge! thank you Ross!!

**Comment by rgordon126** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/6aa4679b-127e-4b4d-8ebc-0466fb3b8d5c?env_id=ibm:yp:us-south "nyu-shopcart-service-s21-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c5243809-8b35-4a28-9d9e-d3fffad0d099 "nyu-shopcart-service-s21") deployed [shopcarts](https://github.com/nyu-devops-team7/shopcarts/releases/tag/deploy-ibm-yp-us-south-rg2846-stern.nyu.edu-dev-20210501-215540 "deploy-ibm-yp-us-south-rg2846-stern.nyu.edu-dev-20210501-215540") to [dev](https://console.bluemix.net/apps/4c97438a-2d8e-4043-97ea-afeca52f1008?env_id=ibm:yp:us-south "ibm:yp:us-south:rg2846@stern.nyu.edu:dev"), including commit d358fb772d2b9e42cc92b04da133c25339a4c691

**Comment by rgordon126** [comment]
IBM Cloud [toolchain](https://cloud.ibm.com/devops/toolchains/6aa4679b-127e-4b4d-8ebc-0466fb3b8d5c?env_id=ibm:yp:us-south "nyu-shopcart-service-s21-toolchain"): [Delivery Pipeline](https://cloud.ibm.com/devops/pipelines/c5243809-8b35-4a28-9d9e-d3fffad0d099 "nyu-shopcart-service-s21") deployed [shopcarts](https://github.com/nyu-devops-team7/shopcarts/releases/tag/deploy-ibm-yp-us-south-rg2846-stern.nyu.edu-prod-20210502-160755 "deploy-ibm-yp-us-south-rg2846-stern.nyu.edu-prod-20210502-160755") to [prod](https://console.bluemix.net/apps/d97eb9f9-ba9d-4eea-ab2c-17189e32ccee?env_id=ibm:yp:us-south "ibm:yp:us-south:rg2846@stern.nyu.edu:prod"), including commit d358fb772d2b9e42cc92b04da133c25339a4c691

**Pull request merged**